logentries/metrics.py

```diff
@@ -639,23 +639,7 @@ def load_json(self, conf):
         # Basic metrics
         metricDict = conf.get(METRIC)
         for item in self.DEFAULTS:
-            try:
                 self.__dict__[item] = metricDict.get(PREFIX + item)
```

Comment:
Is metrics required in the configuration?
If it is not would this cause an error if it is missing?